crates/api_models/src/connector_enums.rs

```diff
@@ -0,0 +1,298 @@
+pub use common_enums::*;
```

Comment:
Can we avoid the wildcard import here, if it's only a few enums that we need here?